src/pages/steps/AdditionalInformationStep.tsx

```diff
@@ -149,6 +152,28 @@ const AdditionalInformationStep = ({
 
   const [tab, setTab] = useState('description');
 
+  const hashHandlerCallback = useCallback(() => {
+    const { hash } = window.location;
+    const newTab = hash.replace('#', '');
+    if (newTab && Object.values(Fields).some((field) => newTab === field)) {
+      setTab(newTab);
+    }
+  }, [setTab]);
+
+  useEffect(() => {
+    hashHandlerCallback();
+
+    window.addEventListener('hashchange', hashHandlerCallback);
+
+    return () => {
+      window.removeEventListener('hashchange', hashHandlerCallback);
+    };
+  }, [hashHandlerCallback]);
+
+  const handleSelectTab = (tab: string) => {
+    window.location.hash = tab;
+  };
```

Comment:
```suggestion
const hash = window.location.hash

useEffect(() => {
  const newTab = hash.replace('#', '');

  if (!newTab || !Object.values(Fields).some((field) => newTab === field)) return;

  setTab(newTab);
}, [hash])

const handleSelectTab = (tab: string) => {
  window.location.hash = tab;
};
```

I think you can write it simpler, something like this.

Reply:
`window.location.hash` is undefined, probably because it gets executed on the server. 
So I used `router` to get the hash instead.